Editor/EditorCore/MenuAction.cs

```diff
@@ -345,7 +376,8 @@ internal virtual bool DoButton(bool isHorizontal, bool showOptions, ref Rect opt
 
             if (iconMode)
             {
-                if (GUILayout.Button(buttonEnabled || !disabledIcon ? icon : disabledIcon, ToolbarGroupUtility.GetStyle(group, isHorizontal), layoutOptions))
+                var bestIcon = icon2x == null ? icon : icon2x;
```

Comment:
same question as above: I'd expect this to be dependent on display resolution, not just loading the highest res possible.